common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -239,7 +245,10 @@ def get_html(self):
             'element_class': self.location.category,
             'ajax_url': self.system.ajax_url,
             'num_inputs': self.num_inputs,
-            'submitted': self.submitted
+            'submitted': self.submitted,
+            'instructions_default': WordCloudFields.instructions.default,
+            'instructions': self.instructions,
+            'display_name': self.display_name
```

Comment:
OK so I'll add a check to see if it differs from the default, like I did with instructions. That way it won't show up unless it's been changed.